conf/cassandra.yaml

```diff
@@ -1625,6 +1625,25 @@ report_unconfirmed_repaired_data_mismatches: false
 # warming of auth caches prior to node completing startup. See CASSANDRA-16958
 # auth_cache_warming_enabled: false
 
+
+#sstable_compressor:
+  # Defines the compression params used to read/write the sstables.  If undefined the default is LZ4 with 16kb chunk
+  #   length. To disable compression set `class_name: org.apache.cassandra.io.compress.NoopCompressor` or
+  #   set the `enable` parameter as `false`
+  #enable=false # defaults to true
+
+  #chunk_length: 16KiB : defaults to 16KiB
+  # min_compress_ratio is only used in snappy and custom implementations.
+  #min_compress_ratio: 0.0 : defaults to 0.0 see CompressionParams.DEFAULT_MIN_COMPRESS_RATIO
+  # type should be one of the `CompressorType` values:
+  #    lz4,  none,  noop,  snappy,  deflate,  zstd, or custom
+  #type: custom
+  # if custom type is specified compressor class and parameters must be provided
+  #compressor:
+  #  class_name: org.apache.cassandra.io.compress.LZ4Compressor # the compressor class
+  #  parameters:
+  #   - param1 : value
+
```

Comment:
would it be better to put this section next to the `flush_compression` setting.

and none of the comments help the user understand that (schema) table-level settings will override this. i think that would help many a new user…